scene/realistic-lighting-and-shadows/src/main/java/com/esri/samples/realistic_lighting_and_shadows/RealisticLightingAndShadowsController.java

```diff
@@ -76,8 +77,20 @@ public void initialize() {
       // set atmosphere effect to realistic
       sceneView.setAtmosphereEffect(AtmosphereEffect.REALISTIC);
 
+      // set a new calendar and add a date and time
+      calendar = new GregorianCalendar(2018, 7, 10, 12, 00, 0);
+      calendar.setTimeZone(TimeZone.getTimeZone("PST"));
+
       // set the time label on the control panel
-      setTimeLabel();
+      sceneView.setSunTime(calendar);
+
+      // tidy string to just return date and time (hours and minutes)
+      dateFormat = new SimpleDateFormat("EEE MMM dd HH:mm");
+      dateFormat.setTimeZone(TimeZone.getTimeZone("PST"));
+      String dateAndTimeTidied = dateFormat.format(calendar.getTime());
+
+      // set a label to display the tidied date and time
+      time.setText(dateAndTimeTidied);
```

Comment:
Perhaps we can call this 'format the string'? We should also name the Label object `timeLabel`, since it helps to identify what kind of class it is.

```suggestion
      // format the string to just return date and time (hours and minutes)
      dateFormat = new SimpleDateFormat("EEE MMM dd HH:mm");
      dateFormat.setTimeZone(TimeZone.getTimeZone("PST"));
      String formattedDateAndTime = dateFormat.format(calendar.getTime());

      // set a label to display the tidied date and time
      timeLabel.setText(formattedDateAndTime);
```